### DISCUSSION TITLE
Include Javascript code into head meta

### DISCUSSION THREAD

**Discussion by kjbtech** [cotes2020/jekyll-theme-chirpy#1639] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I would like to add a javascript code to track visitors on my site.
The more efficient way would be to add it within head meta.
And it is not a Google Analytics code.

In my case, it is from Matomo Analytics.

### Operations you have already tried

I know that I could use includes pattern, according to Jekkyl documentation.
The thing is that copy paste what is it into the include folder on the themes is not the best way to do it.

I think that it could be more efficient to have a way to add javascript file, like we do for tabs, that could be added in head meta for example.

Maybe it is existing or maybe someone already find a way to do it.

Do you?

**Comment by kungfux** [comment, marked as answer]
I think the only way to do this is to override `_includes/head.html` as mentioned in #795.

The alternative would be to add Matomo Analytics to the list of supported analytics providers, as is done for Google Analytics and GoatCounter.

**Reply by kjbtech** [reply]
Thanks for the feedback.
I'm gonna create a new issue to ask for Matomo Analytics support and we'll see!

**Reply by ildoc** [reply]
here's a PR for that #1658

**Reply by kjbtech** [reply]
Amazing thanks!

**Reply by AblabiX** [reply]
> here's a PR for that #1658

when i try to add the matomo id and matomo domain on the _config.yml
dont work, i had to force the code on _includes/head.html

what im doing wrong?
